furusiya/model/maps/generators/forest_generator.py

```diff
@@ -0,0 +1,256 @@
+from attrdict import AttrDict
```

Comment:
Why do we need this here?

Reply:
Using it as a dummy object in _random_walk